File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/MediaCommonConfig.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class MediaCommonConfig{

	@SerializedName("dynamicReadaheadConfig")
	private DynamicReadaheadConfig dynamicReadaheadConfig;

	public void setDynamicReadaheadConfig(DynamicReadaheadConfig dynamicReadaheadConfig){
		this.dynamicReadaheadConfig = dynamicReadaheadConfig;
	}

	public DynamicReadaheadConfig getDynamicReadaheadConfig(){
		return dynamicReadaheadConfig;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/MediaCommonConfig.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class MediaCommonConfig : Parcelable, Serializable {
    @SerializedName("dynamicReadaheadConfig")
    var dynamicReadaheadConfig: DynamicReadaheadConfig? = null
    override fun toString(): String {
        return "MediaCommonConfig{" +
                "dynamicReadaheadConfig = '" + dynamicReadaheadConfig + '\'' +
                "}"
    }
}